src/data/team/people.js

```diff
@@ -15,7 +15,7 @@ import Lyudmil from './images/Lyudmil.png'
 import Maksim from './images/Maksim.png'
 import MaksimR from './images/MaksimR.png'
 import Max from './images/Max.png'
-import Mayumi from './images/Mayumi.png'
+// import Mayumi from './images/Mayumi.png'
```

Comment:
@haritonasty Why do you keep the comments here?

Reply:
I thought about if we need to return her in future ) But, agree, will remove it